com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineCore.cs

```diff
@@ -418,7 +418,43 @@ static bool IsMultiPassStereoEnabled(Camera camera)
             return false;
         }
 
-        Comparison<Camera> cameraComparison = (camera1, camera2) => { return (int) camera1.depth - (int) camera2.depth; };
+#if ENABLE_VR && ENABLE_VR_MODULE
+        static List<XR.XRDisplaySubsystem> displaySubsystemList = new List<XR.XRDisplaySubsystem>();
+        static XR.XRDisplaySubsystem GetXRDisplaySubsystem()
+        {
+            XR.XRDisplaySubsystem display = null;
+            SubsystemManager.GetInstances(displaySubsystemList);
+
+            if (displaySubsystemList.Count > 0)
+                display = displaySubsystemList[0];
+
+            return display;
+        }
+
+        // NB: This method is required for a hotfix in Hololens to prevent creating a render texture when using a renderer
+        // with custom render pass.
+        // TODO: Remove this method and usages when we have proper dependency tracking in the pipeline to know
+        // when a render pass requires camera color as input.
+        internal static bool IsRunningHololens(CameraData cameraData)
+        {
+#if PLATFORM_WINRT
+            if (cameraData.xr.enabled)
+            {
+                var platform = Application.platform;
+                if (platform == RuntimePlatform.WSAPlayerX86 || platform == RuntimePlatform.WSAPlayerARM || platform == RuntimePlatform.WSAPlayerX64)
+                {
+                    var displaySubsystem = GetXRDisplaySubsystem();
```

Comment:
Noticed a error/crash condition.  Not sure how to trigger, but better to be defensive since this is a code path rarely get tested.

1. There's a change `GetXRDisplaySubsystem` return null, so the if condition below should be `if (displaySubsystem != null && !displaySubsystem.displayOpaque) return true;
2. Better rename the helper function as "GetFirstXRDisplaySubsystem" since it's hardcoded to index 0.

Reply:
Thanks and and agree on the suggested changes. 
Addressed in b17c607.